CONTRIBUTING.md

```diff
@@ -85,3 +87,5 @@ For all new Pull Requests the following rules apply:
 - Existing tests should continue to pass.
 - [Tests](docs/AutomatedTests.md) need to be provided for every bug/feature that is completed.
 - Documentation needs to be provided for every feature that is end-user visible.
+- All pull requests must refer to a valid issue number in the repo. For trivial changes refer to
```

Comment:
This would seem to add extra process to dependabot PRs that are unnecessary.
Furthermore, for things like typo fixes, this again seems heavyweight.   Currently line 47 of this file says to use issues for things that are new features (akin to what we use the "enhancement" label for).   If we want to make the process more heavyweight we should probably change the wording there too.